packages/compass-import-export/src/modules/import.ts

```diff
@@ -300,23 +301,36 @@ export const startImport = () => {
       docsWritten: number;
       bytesProcessed: number;
     }) {
-      // for now, call onGuesstimatedDocsTotal() so that the existing progress bar works
       const averageSize = bytesProcessed / docsProcessed;
       const guessedTotal = Math.max(
         docsProcessed,
         Math.ceil(fileSize / averageSize)
       );
-      dispatch(onGuesstimatedDocsTotal(guessedTotal));
-
-      dispatch(onGuesstimatedProgress(docsProcessed, guessedTotal));
-
       dispatch(
         onProgress({
           docsWritten,
           docsProcessed,
           errors: errors.slice(), // make sure it is not the same variable
         })
       );
+      if (!importDone) {
+        // Update the toast with the new progress.
+        openToast(importToastId, {
+          dataTestId: 'import-toast-in-progress',
+          title: `Importing ${fileType} file...`,
+          body: useToastAction({
+            statusMessage: `${docsWritten}/${guessedTotal}`,
```

Comment:
We had a `title` in the previous implementation that would show you what these numbers mean actually as without context they are kinda confusing. Would it make sense to have something similar here? Maybe just use this text as a message instead of just numbers?

Reply:
Updated this to
```
statusMessage={`${docsWritten}/${docsProcessed} document${
  docsWritten !== 1 ? 's' : ''
} written.`}
```
How does that look? Do you think having a `title` on the numbers will still benefit folks?